extensions/wikia/PhalanxII/fallback/Phalanx.class.php

```diff
@@ -321,32 +321,32 @@ static public function addStats( $blockerId, $type ) {
 	 *
 	 * @author Maciej Błaszkowski <marooned at wikia-inc.com>
 	 */
-	static function isBlocked($text, $blockData, $writeStats = true) {
+	static function isBlocked( $text, $blockData, $writeStats = true ) {
 		wfProfileIn( __METHOD__ );
-		$result = array('blocked' => false, 'msg' => '');
+		$result = array( 'blocked' => false, 'msg' => '' );
 		$blockText = $blockData['text'];
 
-		if ($blockData['regex']) {
-			//escape slashes uses as regex delimiter
-			$blockText = str_replace('/', '\/', preg_replace('|\\\*/|', '/', $blockText));
-			if ($blockData['exact']) {
-				//add begining and end anchor only once (user might added it already)
-				if (strpos($blockText, '^') !== 0) {
+		if ( $blockData['regex'] ) {
+			// escape slashes uses as regex delimiter
+			$blockText = str_replace( '/', '\/', preg_replace( '|\\\*/|', '/', $blockText ) );
+			if ( $blockData['exact'] ) {
+				// add begining and end anchor only once (user might added it already)
+				if ( strpos( $blockText, '^' ) !== 0 ) {
 					$blockText = '^' . $blockText;
 				}
-				if (substr($blockText, -1) != '$') {
+				if ( substr( $blockText, -1 ) != '$' ) {
 					$blockText .= '$';
 				}
 			}
 			$blockText = "/$blockText/";
-			if (!$blockData['case']) {
+			if ( !$blockData['case'] ) {
 				$blockText .= 'i';
 			}
-			//QuickFix™ for bad regexes
-			//TODO: validate regexes on save/edit
+			// QuickFix™ for bad regexes
+			// TODO: validate regexes on save/edit
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)